src/NUnitEngine/nunit.engine/Services/DomainManager.cs

```diff
@@ -149,6 +152,38 @@ public void Unload(AppDomain domain)
             new DomainUnloader(domain).Unload();
         }
 
+        public static bool TryGetConfigFileName(string settingsConfigFile, string testFile, string appBaseDir, out string config)
```

Comment:
Because this method is public, it needs XMLDOC comments otherwise it will generate a warning. Can the method be made internal and tested using InternalsVisibleTo?